src/components/Board/components/Column/index.js

```diff
@@ -41,7 +42,10 @@ function Column({
             data-testid={`column-${children.id}`}
           >
             <div {...columnProvided.dragHandleProps}>{renderColumnHeader(children)}</div>
-            {allowAddCard && <CardAdder column={children} onConfirm={onCardNew} />}
+            {allowAddCard &&
+              ((renderCardAdder && renderCardAdder({ column: children, onConfirm: onCardNew })) || (
```

Comment:
Like the other props (e.g.  `renderColumnHeader`), I think this component should only use the `renderCardAdder`. What about to make this decision in the parent?